### DISCUSSION TITLE
测试结果正常么？

### DISCUSSION THREAD

**Discussion by dayunliang** [kkkgo/PaoPao-Pref#7] [提问和帮助]
<img width="605" alt="截屏2024-09-01 20 46 00" src="https://github.com/user-attachments/assets/d80122b1-bf66-4211-972a-585acdc56793">

附件是我刚才结束的测试结果截图，成功率只有25.56% 这正常么？看之前的XD有90多的成功率，改如何改善呢？

**Comment by kkkgo** [comment]
这个跟你的网络环境，性能资源，测试参数都有关系。
非要说怎么改善的话，网络环境尽量减少nat层数，比如docker使用host模式，如果性能遇到瓶颈就多加点资源，调整更宽松的测试参数。

**Reply by dayunliang** [reply]
谢谢回复，昨晚调整了内存又跑了次 PC=100 , 准确率达到了52.67%，并且redis_dns_v2.rdb文件大小达到了217MB,是不是缓存文件越大DNS查询会相对越快？
![Uploading 截屏2024-09-02 08.10.03.png…]()

**Reply by kkkgo** [reply]
并没有一定内存越大查询会越快，但内存越大允许的DNS缓存空间越大，执行查询类似的域名命中率有可能提高，比如缓存了主域名，那么查询子域名会快一点。

**Reply by dayunliang** [reply]
是不是可以理解为准确率越高，DNS查询速度相对越快，总觉得家里的域名查询速度时快时慢

**Reply by kkkgo** [reply]
这不是准确率，这是成功率，如果在给定的测试参数内查询到解析结果定义为成功。跟查询速度关系不大，但可以作为生成缓存或者查询并发性能的参考。

**Reply by dayunliang** [reply]
好的，谢谢

**Reply by biggbuddy** [reply]
我使用默认参数， 10G内存， J4125(host proxmox lxc) , dns 和 perf 都是使用docker host, 成功率51%左右
